pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -43,7 +43,7 @@
 
 # tolerance for numerical errors
 tolerance = 1e-6
-
+MPI_Support = False
```

Comment:
I don't think we need this, since this variable is defined in the try-except block (77-86)
```suggestion
```

Reply:
done!